ambry-api/src/main/java/com/github/ambry/account/Account.java

```diff
@@ -231,6 +231,14 @@ public int getSnapshotVersion() {
     return snapshotVersion;
   }
 
+  public void setStatus(AccountStatus status) {
```

Comment:
Why were these two setters added?

Reply:
I had added them to help with modification of existing Accounts in cache (were being called from AccountInfoMap::updateAccounts()) . But, based on the java docs, it looks like Account and Container classes are supposed to be treated as immutable objects. Changed code to use AccountBuilder to re-build new Account objects while updating accounts.